lib/device/lib/command.js

```diff
@@ -53,7 +53,7 @@ Command.prototype.adb = function(command) {
  * @returns {Command}
  */
 Command.prototype.adbShell = function(command) {
-  return this.adb("shell '" + command + "'");
+  return this.adb("shell '" + command + "'"); // eslint-disable-line
```

Comment:
I'm assuming this is for double quotes. Once we move to ES6, we should remove this in favor of string template.